src/components/themed/CryptoExchangeFlipInputWrapperComponent.js

```diff
@@ -51,6 +56,13 @@ class CryptoExchangeFlipInputWrapperComponent extends React.Component<Props & Th
     )
   }
 
+  onError = (error?: string) => this.setState({ error })
+
+  focusMe = () => {
+    this.onError()
```

Comment:
Why are you calling onError on focus and not only when you actually have an error?